src/batch-client/index.ts

```diff
@@ -114,6 +116,13 @@ export class ZimbraBatchClient {
 		});
 	};
 
+	public addSignature = (options: SignatureInput) =>
```

Comment:
- [ ] Rename `addSignature` to `createSignature` (and put in file in appropriate alphabetical place to match `CreateSignature` request name in API